src/pages/ClusterPage/new.js

```diff
@@ -210,7 +210,7 @@ export default class NewCluster extends Component {
 		let pattern = /^[a-zA-Z0-9]+([-]+[a-zA-Z0-9]*)*[a-zA-Z0-9]+$/;
 		if (provider === 'gke') {
 			// gke cluster names can't start with a number, no capital letters allowed
-			pattern = /^[a-z]+([-]+[a-z0-9]*)*[a-z0-9]+$/;
+			pattern = /^[a-z]+([-a-z0-9]*)*[a-z0-9]+$/;
```

Comment:
Why do we have the last `[a-z0-9]+` condition? That requires two or more characters. And doesn't allow last character to be a `-`. I don't think this is required.

Reply:
Naming convention for GKE:
- Name must be lowercase letters, numbers, and hyphens
- Name must start with a lowercase letter
- Name must end with a lowercase letter or a number.